build.gradle

```diff
@@ -62,7 +62,7 @@ ext {
     commonsIoVersion = '2.13.0'
     commonsNetVersion = '3.9.0'
     curatorVersion = '5.5.0'
-    debeziumVersion = '2.3.0.Final'
+    debeziumVersion = '2.3.3.Final'
```

Comment:
I don't see this version in Maven Central: https://mvnrepository.com/artifact/io.debezium/debezium-embedded

What is their release plan?
Probably we can move on to `2.4` in our `main` and it is is going to be GA for our GA in November?